pkg/analysis/passes/virusscan/virusscan.go

```diff
@@ -45,6 +46,12 @@ var Analyzer = &analysis.Analyzer{
 
 func run(pass *analysis.Pass) (interface{}, error) {
 
+	skip := os.Getenv("SKIP_CLAMAV")
```

Comment:
clamavscan takes a considerable amount of time to run in my system and makes developing quite slow. I am adding this env variable here to skip the whole run validation.